mscore/events.cpp

```diff
@@ -252,6 +252,7 @@ void ScoreView::mouseReleaseEvent(QMouseEvent*)
                   if (editData.startMove == editData.pos && clickOffElement) {
                         _score->deselectAll();
                         _score->update();
+                        mscore->updateInspector();
```

Comment:
Alternatively, we could call `mscore->endCmd()`, which in turn calls `mscore->updateInspector()`, but this may be all that is needed.